deepmd/utils/argcheck.py

```diff
@@ -334,9 +334,7 @@ def descrpt_hybrid_args():
 
 @descrpt_args_plugin.register("se_atten")
 def descrpt_se_atten_args():
-    doc_compressible = (
-        "Whether to build a compressible se_atten that is supported by `dp compress`."
-    )
+    doc_compressible = "Whether to build a compressible `se_atten` that is supported by `dp compress`. Attention: the compressible `se_atten` disables attention layers. Setting it to `False` will fallback to the previous behavior of `se_atten`."
```

Comment:
In principle the model compression happens to the embedding network and is decoupled from the attention layers. I do not understand why one cannot use attention layers when he/she set the this key to True

Reply:
I agree that when using attention layers the acceleration of the model compression may not be as significant as that without attention layers.